docs/style-guide/colors.md

```diff
@@ -37,13 +37,14 @@ export default {
       }, [])
 
     this.$page.colors = {
-      Blues: colors.filter(i => i.includes('blue')),
-      Reds: colors.filter(i => i.includes('red')),
-      Greens: colors.filter(i => i.includes('green')),
-      Yellows: colors.filter(i => i.includes('yellow')),
-      Blacks: colors.filter(i => i.includes('black')),
-      Greys: colors.filter(i => i.includes('grey')),
-      Whites: colors.filter(i => i.includes('white'))
+      Blue: colors.filter(i => i.includes('blue')),
```

Comment:
what is the rhyme reason to the numbers? I would expect them all to be 100-700 but black and gray seem to diverge from this pattern so assuming that's because they reflect the opacity values, but should we just make them all 100-700 to make it easier to remember? Also, should we be using transparency instead of hex for black and gray? Seems like it will be hard to use these values for backgrounds:

![image](https://user-images.githubusercontent.com/5770711/85621556-74f4a980-b633-11ea-9f10-6b1babe06cd6.png)

Reply:
there is no particular reason, they are numbered lowest to highest of the needed color. Because there has never been a need for other yellows or greens there is less of them. Black and grey were left alone. Grey's could be made into a scale and could have been from the beginning I suppose but we're also not touched during this. Blacks I believe should remain the same because they are the only color changed by opacity.

Reply:
Initially the idea was only change new colors but after discussing with design further we’ve decided to rename the greys to follow the other colors. Blacks will remain titled with their opacity.